docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[connecting-to-cloud]]
+=== Send {ls} Output to Elastic Cloud
```

Comment:
I think we should have consistent terminology here. Maybe "Sending Data to Elasticsearch Service"?

Reply:
I changed this, and it's showing in the preview.  

Now that I look at it, I'm wondering if "Send Logstash Output to Elastic Cloud"  or even "Send Logstash Output to the Cloud" might be more eye-catching because Cloud is such a popular buzzword.  Please let me know what you think.